samples/extauthz/README.md

```diff
@@ -4,7 +4,7 @@
 as an example of integrating custom authorization system into Istio.
 
 The Ext Authz server supports authorization check request using either HTTP (port 8000) or gRPC v2/v3 (port 9000) API and
-will allow the request if it includes the header `x-ext-authz: allow`.
+will allow the request if it includes the header `x-ext-authz: allow`. For TCP requests, the Ext Authz server will allow requests from source account `a`.
```

Comment:
This is not specific to TCP, it also allows HTTP. And make it clear the the service account (not source account) `a` is just a default value for testing and can be configured explicitly with the flag `-allow_service_account`